examples/java/README.md

```diff
@@ -0,0 +1,36 @@
+# Java Example
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Kotlin/kotlinx-benchmark)
+
+## Project Structure
+
+Inside of this example, you'll see the following folders and files:
+
+```
+/
+├── build.gradle
+└── src/
+    └── main/
+        └── java/
+            └── test/
+                └── SampleJavaBenchmark.java
+```
+
+## Tasks
+
+All tasks can be run from the root of the project, from a terminal:
+
+| Task Name | Action |
+| --- | --- |
+| `gradle assembleBenchmarks` | Generate and build all benchmarks in the project |
+| `gradle benchmark` | Execute all benchmarks in the project |
+| `gradle mainBenchmark` | Execute benchmark for 'main' |
```

Comment:
Could you please specify what 'main' stands for in the task description. It can be done in `createJvmBenchmarkExecTask` function in JvmTasks.kt. Basically, 'main' is the name of the source set registered as a benchmark target.